cellmlmanip/model.py

```diff
@@ -308,8 +308,38 @@ def get_definition(self, variable):
         return defn
 
     def get_value(self, variable):
-        """Returns the evaluated value of the given variable's RHS."""
-        return float(self.graph.nodes[variable]['equation'].rhs)
+        """
+        Returns the evaluated value of the given variable, as a float.
+
+        For state variables, this returns the initial value. For variables that depend on other variables this
+        recursively evaluates any dependencies (again at the initial state). Zero is returned for variables without a
+        definition (e.g. for the free variable).
+        """
+        return self._get_value(variable)
+
+    def _get_value(self, variable, evaluated=None):
+        """Internal method to implement :meth:`get_value()`."""
+
+        # State? Then return initial value
+        if variable in self._ode_definition_map:
+            return float(variable.initial_value)
+
+        # Get RHS and evaluate
+        expr = self.graph.nodes[variable]['equation']
```

Comment:
Ooh, actually, we should use
```suggestion
        expr = self._var_definition_map.get(variable)
```
for this now!